spec/rspec/core/hooks_spec.rb

```diff
@@ -145,6 +145,34 @@ def hook_collection_for(position, scope)
       end
     end
 
+    describe "#before(:all)" do
+      it "stops running subsequent after hook when an error is encountered" do
```

Comment:
I'd put this next to `it "runs subsequent hooks of the same type when an error is encountered so all cleanup can complete" do`.